clusterdock/utils.py

```diff
@@ -95,3 +95,29 @@ def join_url_parts(*parts):
     examples of why urllib.parse.urljoin is insufficient for what we want to do.
     """
     return '/'.join([piece.strip('/') for piece in parts])
+
+
+def version_tuple(version, default=None):
+    """
+    Convert a version string or tuple to a tuple.
+    Will return (major, minor, release) kind of format.
+    """
+    if isinstance(version, str):
+        return tuple(int(x) for x in version.split('.'))
+    elif isinstance(version, tuple):
+        return version
+    else:
```

Comment:
I think we can just leave out the ``default`` argument. I don't think people would need to change that default in this function very often and Python functions return `None` implicitly if we don't return anything explicit. Same comment for the function below.

Reply:
Done